tests/unit/hive_metastore_client/test_hive_metastore_client.py

```diff
@@ -472,3 +473,47 @@ def test_bulk_drop_partitions_with_errors(
             )
 
         assert mock_drop_partition.call_count == len(partition_list)
+
+    @mock.patch.object(HiveMetastoreClient, "get_partition_values", return_value=[])
+    @mock.patch.object(
+        HiveMetastoreClient, "get_partition_keys_objects", return_value=[]
+    )
+    def test_get_partition_values_from_table_with_partitioned_table(
+        self,
+        mocked_get_partition_keys_objects,
+        mocked_get_partition_values,
+        hive_metastore_client,
+    ):
+        # arrange
+        table_name = "table_name"
+        database_name = "database_name"
+        mocked_partition_values_response = Mock()
+        mocked_partition_values = []
+
+        mocked_partition_values_partition_a = Mock()
+        mocked_partition_values_partition_a.row = ["partition_a"]
+        mocked_partition_values.append(mocked_partition_values_partition_a)
+        mocked_partition_values_partition_b = Mock()
+        mocked_partition_values_partition_b.row = ["partition_b"]
+        mocked_partition_values.append(mocked_partition_values_partition_b)
+
+        mocked_partition_values_response.partitionValues = mocked_partition_values
```

Comment:
no biggie. In my view its clearer:
```suggestion

        mocked_partition_values = []
        
        mocked_partition_values_partition_a = Mock()
        mocked_partition_values_partition_a.row = ["partition_a"]
        mocked_partition_values.append(mocked_partition_values_partition_a)
        
        mocked_partition_values_partition_b = Mock()
        mocked_partition_values_partition_b.row = ["partition_b"]
        mocked_partition_values.append(mocked_partition_values_partition_b)

        mocked_partition_values_response.partitionValues = mocked_partition_values
```